api/python/notebooks/README.md

```diff
@@ -26,15 +26,10 @@ as it will keep you from running out of RAM.
   $ python -m venv ./venv
   $ source ./venv/bin/activate
 ```
-2. Install the `cell_census` package using `pip`:
+2. Install the required dependencies:
 ```shell
-  $ pip install -U cell-census
+  $ pip install -r cell-census/api/python/notebooks/requirements.txt
```

Comment:
while `-U` is not necessary for a first-time installation, I don't think there's any harm in keeping it; if a user repeats the command later they'll get a new cell-census. I'm 50/50 on whether to keep it. :)

Reply:
Let's keep it.